### PR TITLE
Update Plugin parent pom to 2.12

### PR DESCRIPTION
**Pull request by oleg-nenashev**
This change updates Parent POM to the modern version. Failure on FindBugs errors is disabled since there are 17 bugs requiring some love.

@reviewbybees and @recena

### CODE REVIEW

**Comment by oleg-nenashev** [comment]
As always, had to switch the PR builder to hi-speed

File: pom.xml [hunk 1]
```diff
@@ -5,7 +5,7 @@
     <parent>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
-        <version>1.532.3</version>
+        <version>2.11</version>
```
Comment (recena) [inline pom.xml]: Maybe `2.12`?

**Comment by recena** [comment]
🐝

File: pom.xml (hunk 1 above)
Comment (oleg-nenashev) [inline pom.xml]: Makes sense

**Comment by oleg-nenashev** [comment]
@reviewbybees done
Acting further as a maintainer

**Comment by oleg-nenashev** [comment]
Local build for 2.12 has successfully passed. I'll merge the change when the build passes

**Comment by oleg-nenashev** [comment]
Going forward with merge. FindBugs is another story :(

**Pull request merged**